src/containers/ProfilePage/__snapshots__/index.test.jsx.snap

```diff
@@ -94,6 +96,7 @@ exports[`renders correctly 1`] = `
                     Your nationality
                   </label>
                   <input
+                    className=""
```

Comment:
you want these to go away. please delete the OR "" in your code

Reply:
I deleted them, but with git reset but sorry it seems that did work. It's done!